parcels/field.py

```diff
@@ -279,6 +279,8 @@ def from_netcdf(cls, filenames, variable, dimensions, indices=None, grid=None,
         dimension_filename = dimension_filename if dimension_filename else filenames[0]
         if indices is None:
             indices = {}
+        else:
+            indices = indices.copy()
```

Comment:
Or as a one-liner?
```
indices = {} if indices is None else indices.copy()
```

Or is that too confusing? If so, then perhaps also add some comments here?